configs/cornernet/cornernet_hourglass104_8x6_210e_coco.py

```diff
@@ -0,0 +1,117 @@
+        loss_offset=dict(type='SmoothL1Loss', beta=1.0, loss_weight=1)))
+# data settings
+dataset_type = 'CocoDataset'
+data_root = 'data/coco/'
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+train_pipeline = [
+    dict(type='LoadImageFromFile', to_float32=True),
+    dict(type='LoadAnnotations', with_bbox=True),
+    dict(
+        type='PhotoMetricDistortion',
+        brightness_delta=32,
+        contrast_range=(0.5, 1.5),
+        saturation_range=(0.5, 1.5),
+        hue_delta=18),
+    dict(
+        type='RandomCenterCropPad',
+        crop_size=(511, 511),
+        ratios=(0.6, 0.7, 0.8, 0.9, 1.0, 1.1, 1.2, 1.3),
+        test_mode=False,
+        test_pad_mode=None,
+        **img_norm_cfg),
+    dict(type='Resize', img_scale=(511, 511), keep_ratio=False),
+    dict(type='RandomFlip', flip_ratio=0.5),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='DefaultFormatBundle'),
+    dict(type='Collect', keys=['img', 'gt_bboxes', 'gt_labels']),
+]
+test_pipeline = [
+    dict(type='LoadImageFromFile', to_float32=True),
+    dict(
+        type='MultiScaleFlipAug',
+        scale_factor=1.0,
+        flip=False,
+        transforms=[
+            dict(
+                type='RandomCenterCropPad',
+                crop_size=None,
+                ratios=None,
+                border=None,
+                test_mode=True,
+                test_pad_mode=['logical_or', 127],
+                **img_norm_cfg),
+            dict(type='RandomFlip'),
+            dict(type='Normalize', **img_norm_cfg),
+            dict(type='ImageToTensor', keys=['img']),
+            dict(
+                type='Collect',
+                keys=['img'],
+                meta_keys=('filename', 'ori_shape', 'img_shape', 'pad_shape',
+                           'scale_factor', 'flip', 'img_norm_cfg', 'border')),
+        ])
+]
+data = dict(
+    samples_per_gpu=6,
+    workers_per_gpu=3,
+    train=dict(
+        type=dataset_type,
+        ann_file=data_root + 'annotations/instances_train2017.json',
+        img_prefix=data_root + 'train2017/',
+        pipeline=train_pipeline),
+    val=dict(
+        type=dataset_type,
+        ann_file=data_root + 'annotations/instances_val2017.json',
+        img_prefix=data_root + 'val2017/',
+        pipeline=test_pipeline),
+    test=dict(
+        type=dataset_type,
+        ann_file=data_root + 'annotations/instances_val2017.json',
+        img_prefix=data_root + 'val2017/',
+        pipeline=test_pipeline))
+evaluation = dict(interval=1, metric='bbox')
+# training and testing settings
+train_cfg = None
+test_cfg = dict(
+    corner_topk=100,
+    local_maximum_kernel=3,
+    distance_threshold=0.5,
+    score_thr=0.05,
+    max_per_img=100,
+    nms_cfg=dict(type='soft_nms', iou_threshold=0.5, method='gaussian'))
+# optimizer
+optimizer = dict(type='Adam', lr=0.0005)
+optimizer_config = dict(grad_clip=dict(max_norm=35, norm_type=2))
```

Comment:
Is `grad_clip` necessary for CornerNet? If not, remove it.

Reply:
I didn't implement any experiment without `grad_clip`, so I think it's better to keep it.